pkg/sfu/downtrack.go

```diff
@@ -349,6 +341,13 @@ func NewDownTrack(params DowntrackParams) (*DownTrack, error) {
 	})
 
 	if d.kind == webrtc.RTPCodecTypeVideo {
+		if delay := params.PlayoutDelayLimit; delay.GetEnabled() {
```

Comment:
audio's controller will be created but never used as no playout delay extension negotiated for audio track, remove it